go.sum

```diff
@@ -1,5 +1,3 @@
-github.com/davecgh/go-xdr v0.0.0-20161123171359-e6a2ba005892 h1:qg9VbHo1TlL0KDM0vYvBG9EY0X0Yku5WYIPoFWt8f6o=
```

Comment:
We still need this, can you revert go.sum to the master branch version?

Reply:
This was automatically getting removed from go.sum. 
Reverted back go.sum.